src/Illuminate/Foundation/resources/exceptions/renderer/components/frame-code.blade.php

```diff
@@ -10,6 +10,6 @@ class="text-sm rounded-b-lg bg-neutral-50 border-t border-neutral-100 dark:bg-ne
         editor
         :starting-line="max(1, $highlightedLine - 5)"
         :highlighted-line="min(5, $highlightedLine - 1)"
-        class="overflow-x-auto"
+        class="overflow-x-auto [&>pre]:w-fit [&>pre]:min-w-full"
```

Comment:
Can we add those classes here instead?
https://github.com/laravel/framework/blob/5e1b8fee6f0b742dbdb085aa22d54b201170e202/src/Illuminate/Foundation/resources/exceptions/renderer/scripts.js#L63-L65

They should not be added when `truncate` is `true`.

```javascript
this.addClassToHast(node, ['bg-transparent!', truncate ? 'truncate' : 'w-fit min-w-full']);
```

Reply:
sure thing!